scripts/createComponent.js

```diff
@@ -146,21 +172,41 @@ const linkPackages = async () => {
   console.log(`⚠️  Please ensure you that it is exported in packages/ltht-react/src/index.ts`)
 }
 
-const init = () => {
+const init = async () => {
   if (fs.existsSync(path.join(folderName))) {
     console.log(
       `❌ The ${componentType} component '${componentName}' already exists Please use a different name or delete the existing folder`
     )
     return false
   }
-  mkdirp(`${folderName}/src`).then(() => {
-    indexScript()
-    packageJson()
-    readmeScript()
-    tsconfigScript()
-    linkPackages()
-  })
+  await fs.promises.mkdir(`${folderName}/src`, { recursive: true })
+  await fs.promises.mkdir(`${folderName}/src/organisms`, { recursive: true })
+  await fs.promises.mkdir(`${folderName}/src/molecules`, { recursive: true })
+  await fs.promises.mkdir(`${folderName}/src/atoms`, { recursive: true })
```

Comment:
I wouldn't bother making the `atoms`, `molecules` and `organisms` folders, as it's not guaranteed that all components will include all three.

Just the `src` folder should be enough